packages/portfolio-contract/src/type-guards.ts

```diff
@@ -326,6 +326,14 @@ export const makeFlowStepsPath = (parent: number, id: number) => [
   'steps',
 ];
 
+export const makeFlowStepPath = (parent: number, id: number, step: number) => [
```

Comment:
this isn't a type guard. I see it's following the pattern of the other path makers. Please leave a TODO to move all the path makers to portfolio-api package. They can be used by clients too.